_source/_posts/2023-06-08-identity-developer-podcast.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: blog_post
+title: "Unlock identity expertise with our new developer podcast!"
+author: edunham
+by: advocate
+communities: [devops]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/identity-developer-podcast/podcast.jpg 
+type: awareness|conversion
+---
+
+Do you enjoy podcasts? I find that, like audiobooks, podcasts can transform boring tasks into an opportunity for entertainment and education. 
+
+I'm delighted to announce a new offering from Okta's Workforce Developer Advocacy Team: our monthly podcast! 
+
+The audio version of the Workforce Identity Developer Podcast is available on [Casted](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f), where you can also explore Auth0's ongoing [Authorization in software podcast](https://authorizationinsoftware.auth0.com/public/49/Authorization-in-Software-f9b69587) and [Identity Unlocked](https://identityunlocked.auth0.com/public/49/Identity%2C-Unlocked.--bed7fada) podcast as well.
```

Comment:
I might consider adding some h2 and use some reinforcing keywords for better SEO here, such as (also a chance to use some of the other titles that we liked)

```
## Listen and learn with the workforce identity podcasts

The audio version...
```